content/en/news/releases/1.7.x/announcing-1.7.5/index.md

```diff
@@ -28,3 +28,7 @@ This release contains bug fixes to improve robustness. This release note describ
 - **Fixed** an issue causing a short spike in errors during in place upgrades from Istio 1.6 to 1.7. Previously, the xDS version would be upgraded automatically from xDS v2 to xDS v3. This caused downtime with upgrades from Istio 1.6 to Istio 1.7. This has been fixed so that these upgrades no longer cause downtime. Note that, as a trade off, upgrading from Istio 1.7.x to Istio 1.7.5 still causes downtime in any existing 1.6 proxies; if you are in this scenario you may set the `PILOT_ENABLE_TLS_XDS_DYNAMIC_TYPES` environment variable to false in Istiod to retain the previous behavior. ([Issue #28120](https://github.com/istio/istio/issues/28120))
 
 - **Fixed** missing listeners on a VM when the VM sidecar is connected to `istiod` but a `WorkloadEntry` is registered later. ([Issue #28743](https://github.com/istio/istio/issues/28743))
+
+### Upgrade Notice
+
+When upgrading your Istio data plane from 1.7.x (where x < 5) to 1.7.5 or newer, you may observe connectivity issues between your gateway and your sidecars or among your sidecars with 503 errors in the log. This happens when 1.7.5+ proxies send HTTP 1xx or 204 response codes with headers that 1.7.x proxies reject. To fix this, upgrade all your proxies (gateways and sidecars) to 1.7.5+ as soon as possible. ([Issue 29427](https://github.com/istio/istio/issues/29427), [More information](https://github.com/istio/istio/pull/28450))
```

Comment:
Is this considered a bug or working as intended? Do we have plans to address this?

Reply:
My understanding is this is how it is until it is addressed in envoy - https://github.com/envoyproxy/envoy/issues/13868

cc @Monkeyanator to correct me if I'm wrong.

Reply:
You're correct. Think we're stuck with this issue on upgrades from 1.7.4- to all newer versions without an envoy fix